pkg/skuba/actions/auth/auth.go

```diff
@@ -186,7 +186,7 @@ func doAuth(authReq request) (*response, error) {
 	ctx := oidc.ClientContext(context.Background(), client)
 	provider, err := oidc.NewProvider(ctx, authReq.IssuerURL)
 	if err != nil {
```

Comment:
Is this everything we can do? Like guess the issue? Dont we have any kind of info coming from the error so we can try to better guess the actual issue? No connection VS insecure its a very different issue so I would expect the error to point to what is wrong*

* knows nothing of go errors or the oidc lib :)

Reply:
I think we can always improve later and evolve this kind of errors, but for the ratio effort/result I think this is good to have and then see if there's need to improve further.

Reply:
Dont get me wrong, presenting solutions to an user instead of plainly telling the error is always good, but Im wondering if it would be better to spend a bit more time and fine tune this rather than blindly guessing what could be wrong.

Reply:
Yes, but this would to the same logic as `oidc.NewProvider` does.

I could investigate is it possible to check the certificate error.

Reply:
having a look at this on the net/http pkg, seems that we would need to regex match the error message as both would come back as `Error`.

Pretty poor error handling IMO, doesnt really merits to spend anymore time into a flaky solution :/

Reply:
go 1.13 have an enhance on the error handling.